docs/source/install/rhel6.rst

```diff
@@ -12,16 +12,9 @@ codifies the instructions below.
 
 .. warning ::
 
-    RHEL 6 doesn't seem to ship with libffi-devel which is a dependency for |st2|.
-    If that is the case, find a version of libffi-devel compatible with libffi on the box.
-    For example,
-
-    .. code :: bash
-
-      [ec2-user@ip-172-30-0-79 ~]$ rpm -qa libffi
-      libffi-3.0.5-3.2.el6.x86_64
-
-      sudo yum localinstall -y ftp://fr2.rpmfind.net/linux/centos/6.7/os/x86_64/Packages/libffi-devel-3.0.5-3.2.el6.x86_64.rpm
+    Your RHEL 6 setup might not have *server-optional* repository enabled which is required
+    for *libffi-devel* dependency. Please consult to the [documentation](https://access.redhat.com/solutions/265523)
+    and enable the repository.
```

Comment:
I like this more, It will be good to tell a user how to check if it's enabled or not. But I am thinking: it's a lot of tribal knowledge, guys who are setting this up will be like Anthony who is smart but may well not know these little details, and now that we figure it, let's share. How about combining the two: 

Your RHEL 6 setup might not have *server-optional* repository enabled which is required for *libffi-devel* dependency. Please consult to the [documentation](https://access.redhat.com/solutions/265523) and enable the repository. Or find and install a version of libffi-devel compatible with libffi on the box: 
```
     # Check the version of libffi
      $ rpm -qa libffi
      libffi-3.0.5-3.2.el6.x86_64
      
      # Install the correspondent versoin of libffi-devel, 3.0.5 in this example
      sudo yum localinstall -y ftp://fr2.rpmfind.net/linux/centos/6.7/os/x86_64/Packages/libffi-devel-3.0.5-3.2.el6.x86_64.rpm
```